### ISSUE TITLE
R key words (and some braces) trigger error in Script window

### ISSUE THREAD

**Issue by lloyddewit** [IDEMSInternational/R-Instat#8285]
Bug described in email exchange below:

From: Wezzie
Sent: 27/04/23

An example of curly bracket error is the same one that I sent in the previous for loops email.

For (b), in your code, some months had abnormal values for Tmin and Tmax (too low or too high). In that case, those values were made missing. For example, code used in the script for Beira station did not run in R-instat but the code below was able to run in R-instat.

```
mozambique_s_daily$Tmin[(mozambique_s_daily$station == "Beira" & 
                          mozambique_s_daily$date == as.Date("2021/06/21"))] <- NA
```

From: Wezzie
Sent: 21/04/23

a.) I have tried running both Mozambique_2023.R and qc.R scripts. I have noted that I am still finding it hard to run "for loops"' and "if statements". However, I have noted that the case is different for "ifelse" functions. 

For instance, I tried to run a simple for loop code below.

                              for (i in c(-8, 9, 11, 45)) {
                                  print(i)
                                }
From the simple code above, I got an error message ('token has an unexpected type). This is the same error I got after running a simple if statement code below. 

                            if (2==1) {print("1")
                            } else if (2==2) {print("2")
                            } else print("3")

The error is also the same for the codes in the two scripts. Initially, I thought R-instant might have a problem with the curly bracket, but I have noted that the case is the same even if the curly brackets are absent in the code. Could it be that there is a problem in the back end of R-instat (which I am failing to trace) causing problems when running for loops and if statements? 

b). In the qc.R script, I found it hard to run the scripts which made some months missing. I got an error that a closing bracket has no corresponding bracket, yet the code is just fine and all open brackets have corresponding closing brackets. Even if I add or remove another set of brackets, the same error is still appearing.

**Comment by lloyddewit** [comment]
Part a is fixed in PR #8461.
For part b, I sent Wezzie the following email on 09/08/23:

> I am unable to recreate part b because I do not the have the necessary data files. If you have any simple code that triggers the same error, then please could you share it so I can test? Alternatively, you could wait for the next public release and then check if the problem still exists.

**Comment by lloyddewit** [comment]
@rdstern I think Wezzie is unlikely to send the data we need to test part b. Can we close this issue?

**Comment by lloyddewit** [comment]
I cannot recreate this and I am fairly certain it is fixed by #8746 so I will close this issue.
